pkg/kubelet/volume_host.go

```diff
@@ -270,6 +270,20 @@ func (kvh *kubeletVolumeHost) GetNodeLabels() (map[string]string, error) {
 	return node.Labels, nil
 }
 
+func (kvh *kubeletVolumeHost) GetAttachedVolumes() (map[v1.UniqueVolumeName]string, error) {
+	node, err := kvh.kubelet.GetNode()
```

Comment:
will it cause extra API calls to get node object?

Reply:
It uses node lister:

```
if kl.kubeClient == nil {
	return kl.initialNode(context.TODO())
}
return kl.nodeLister.Get(string(kl.nodeName))
```